cfme/automate/explorer/method.py

```diff
@@ -220,19 +331,36 @@ def tree_path(self):
 
     def create(
             self, name=None, display_name=None, location='inline', script=None, data=None,
-            cancel=False, validate=True):
+            cancel=False, validate=True, repository=None, playbook=None, machine_credential=None,
+            hosts=None, max_ttl=None, escalate_privilege=None, verbosity=None,
+            playbook_input_parameters=None):
         add_page = navigate_to(self, 'Add')
-        fill_dict = {
-            k: v
-            for k, v in {
-                'name': name,
-                'display_name': display_name,
-                'location': location,
+        wait_for(lambda: add_page.location.is_displayed, delay=0.5, num_sec=2)
```

Comment:
**Optional**
Consider using wait_for_view boolean arg for navigate_to(), if `add_page.is_displayed` is sufficient here instead of a specific widget's is_displayed